### DISCUSSION TITLE
Disabling Spellcheck

### DISCUSSION THREAD

**Discussion by mattdogoodnow** [ueberdosis/tiptap#3473] [General Questions] [answered]
Hey everyone! I'm using tip tap and am wondering how I might disable spellcheck. I tried [this](https://github.com/ueberdosis/tiptap/issues/76), but it didn't work for me. 

I'm not a strong coder, sooooo it might be ignorance. But I checked with my strong coding friend and he also said it wasn't possible. Wanted to reach out! Thanks :)

**Comment by mattdogoodnow** [comment, marked as answer]
I reached out to humans@tiptap.dev and they replied soooo fast: 

this worked for me: 

`<EditorContent editor={editor} spellCheck={false} />`

**Comment by benbucksch** [comment]
Set `spellcheck="false"` on a container div element. This worked for me.
See https://developer.mozilla.org/de/docs/Web/HTML/Global_attributes/spellcheck
